app/schema_validation/__init__.py

```diff
@@ -62,6 +63,24 @@ def validate_schema_date_with_hour(instance):
     return True
 
 
+@format_checker.checks("datetime_schedule_job", raises=ValidationError)
+def validate_schema_date_for_job(instance):
+    max_hours = current_app.config["JOBS_MAX_SCHEDULE_HOURS_AHEAD"]
+    if isinstance(instance, str):
+        try:
+            dt = iso8601.parse_date(instance).replace(tzinfo=None)
+            if dt < datetime.utcnow():
+                raise ValidationError("datetime cannot be in the past")
+            if dt > datetime.utcnow() + timedelta(hours=max_hours):
+                raise ValidationError(f"datetime can only be up to {max_hours} hours in the future")
+        except ParseError:
+            raise ValidationError(
+                "datetime format is invalid. It must be a valid ISO8601 date time format, "
+                "https://en.wikipedia.org/wiki/ISO_8601"
```

Comment:
I find it a bit odd to give a Wikipedia link but another error message in this file does that